packages/cli/lib/services/local-integration.service.ts

```diff
@@ -81,7 +82,15 @@ class IntegrationService implements IntegrationServiceInterface {
             } catch (err: any) {
                 // TODO merge this back with the main integration service
                 if (err instanceof ActionError) {
-                    return { success: false, error: err, response: null };
+                    return {
+                        success: false,
+                        error: {
+                            type: err.type,
+                            payload: err.payload || {},
+                            status: 500
```

Comment:
is 500 what we always return? Doesn't NangoError default to 500?

Reply:
Yes it's the default, unless the type is known. 
I just followed the pattern in Line 80, which is returning 500 by default. Is it okay?